### PR TITLE
Prepare release 2.2.6

### PR DESCRIPTION
**Pull request by Autophagy**

### CODE REVIEW

File: blackbox/docs/release_notes/2.2.6.txt [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. _version_2.2.6:
+
+=============
+Version 2.2.6
+=============
+
+Released on 2018/01/15.
```
Comment (matriv) [inline blackbox/docs/release_notes/2.2.6.txt]: Please update

File: blackbox/docs/release_notes/2.2.6.txt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+- Fixed an issue that could cause ``INSERT INTO`` statements into partitioned
+  tables to not work correctly. This only occurred if a ``query`` instead of
+  the ``VALUES`` clause was used.
+
+- Fixed the evaluation of JavaScript user defined functions that caused CrateDB
+  to crash because of an unhandled assertion when providing the UDF with
+  EcmaScript 6 arrow function syntax (``var f = (x) => x;``).
+
+- Fixed an issue where batch operations executed using the postgres protocol
```
Comment (matriv) [inline blackbox/docs/release_notes/2.2.6.txt]: using the ``PostgreSQL wire`` protocol

File: blackbox/docs/release_notes/2.2.6.txt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  returned 0 as row count, even though the actual row count was different.
+
+- Fixed a bug which could cause job entries in `sys.jobs` not being removed
+  when a connection error occurred while transferring the results of the job
```
Comment (matriv) [inline blackbox/docs/release_notes/2.2.6.txt]: while ``sending`` the results.... ``to the client``.

File: core/src/main/java/io/crate/Version.java [hunk 4]
```diff
@@ -41,7 +41,7 @@
 
 
     public static final boolean SNAPSHOT = false;
-    public static final Version CURRENT = new Version(2020599, SNAPSHOT, org.elasticsearch.Version.V_5_5_2_UNRELEASED);
+    public static final Version CURRENT = new Version(2020699, SNAPSHOT, org.elasticsearch.Version.V_5_5_2_UNRELEASED);
```
Comment (emilywoods) [inline core/src/main/java/io/crate/Version.java]: How is this version number generated? (`2020699`)

Comment (Autophagy) [reply core/src/main/java/io/crate/Version.java]: The ID numbers are formatted as ABBCCDD where:
A is the major version (here 2)
BB is the minor version (here 02)
and CC is the revision (here being bumped from 05 to 06)
DD is the beta or release candidate indicator. < 50 are Beta builds, < 99 are Release Candidates and 99 is a release. We typically don't change this number

Comment (emilywoods) [reply core/src/main/java/io/crate/Version.java]: Ahh I see, thanks for explaining!

**Pull request merged**